kde-frameworks/kapidox/kapidox-5.21.0.ebuild

```diff
@@ -4,16 +4,16 @@
 
 EAPI=6
 
-KDE_DEBUG="false"
-KDE_DOXYGEN="false"
-KDE_TEST="false"
 DISTUTILS_SINGLE_IMPL="true"
 PYTHON_COMPAT=( python2_7 )
-inherit kde5 distutils-r1
```

Comment:
Super correct means we loose the uniform handling of kde-frameworks packages. ;)